index.html

```diff
@@ -4,6 +4,10 @@
 julia_version: v1.1
 ---
 
+<meta property="og:title" content="The Julia Language"/>
+<meta property="og:image" content="http://www.julialang.org/images/logo.png"/>
```

Comment:
Should we use the image that we're using for the repo (the top image from https://discourse.julialang.org/t/open-graph-image-for-the-julia-repository/23384/11? Would have to be added to the images/ folder in this repo). Also this should probably go into https://github.com/JuliaLang/www.julialang.org/blob/aef767c6c5070f8acea7e44aadc20013752dfe57/_includes/head.html.

Reply:
Agreed, would look a lot better that way - thanks for the suggestion!  Will make the changes.